desktop/apps/consign/client/reducers.js

```diff
@@ -308,6 +309,13 @@ function submissionFlow (state = initialState, action) {
         user: action.payload.user
       }, state)
     }
+    case actions.UPDATE_USER_PHONE: {
+      return u({
+        user: {
+          phone: action.payload.newPhone
```

Comment:
Why not just 'phone' in both places? dispatch(updatePhone({ payload: phone }) -- sorry, on phone 😊

Reply:
I wanted to be explicit with `USER_PHONE` since most of the fields we update are on a `submission`, but I'm not attached to `newPhone`. 😄